config/locales/en.yml

```diff
@@ -1317,7 +1317,8 @@ en:
       build_failed: "Unable to create the repository with the selected configuration. %{reason}"
       empty_repository: "The repository exists, but is empty. It does not contain any revisions yet."
       exists_on_filesystem: "The repository directory exists already on filesystem."
-      filesystem_access_failed: "An error occurred while accessing the repository on filesystem: %{message}."
+      filesystem_access_failed: "An error occurred while accessing the repository on filesystem: %{message}"
+      deletion_failed: "The repository could not be deleted on disk due to lacking permissions. Please contact your administrator to resolve this issue."
```

Comment:
Correct me if I am wrong, because I sometimes get those pronouns wrong, but shouldn't it be *deleted* ***from*** *disk*?